plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -94,7 +94,7 @@ def build_plugin_wheel(plugin: str, tmp_wheel_dir: str) -> str:
     log.info(f"Build wheel for {plugin}, save wheel to {tmp_wheel_dir}.")
     subprocess.check_output(
         f"python setup.py sdist bdist_wheel && cp dist/*.whl {tmp_wheel_dir}",
-        # shell=True,
+        shell=True,  # nosec
```

Comment:
I am going to push back on this one. I don't think there is a reason to use shell=True here.

Reply:
will fix this! :)